modules/logging-loki-gui-install.adoc

```diff
@@ -14,43 +14,39 @@ To install and configure logging on your {product-title} cluster, additional Ope
 
 .Procedure
 
-. Install the `Loki Operator`:
+. To install the `Loki Operator` in the {product-title} web console, click *Operators* -> *OperatorHub*.
 
-.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+. Type `Loki Operator` in the filter by keyword box. Choose *Loki Operator* from the list of available Operators and click *Install*.
 
-.. Type `Loki Operator` in the filter by keyword box. Choose *Loki Operator* from the list of available Operators and click *Install*.
-+
 [NOTE]
```

Comment:
The `+` needs to be here I think, and the same for the following steps where it was removed. See https://67696--docspreview.netlify.app/openshift-enterprise/latest/logging/cluster-logging-loki#logging-loki-gui-install_cluster-logging-loki